City-of-Helsinki/drupal-helfi-platform-config
# UHF-8616 translatable menu link uri D10

# [UHF-8616](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8616)
The translatable_menu_link_uri contrib module does not have D10 compatibility yet

## What was done
Created a patch based on upgrade status module report

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-8616`
* Run `make drush-updb drush-cr`

## How to test
- composer require drupal/upgrade_status
- drush en upgrade_status
- drush us-a translatable_menu_link_uri

The report should say everything is ok.
(optional) Edit and save menu link. Should work normally.



[UHF-8616]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8616?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
